hs_core/models.py

```diff
@@ -1379,12 +1379,12 @@ class MyResourceContentType(pages.Page, hs_core.AbstractResource):
     content_object = generic.GenericForeignKey('content_type', 'object_id')
 
     def delete(self, using=None):
+        from hydroshare import hs_bagit
```

Comment:
Yes, circular imports is the reason. If I put it on the top, I will get ImportError.